spdx/writers/jsonyamlxml.py

```diff
@@ -475,15 +477,25 @@ def create_document(self):
         self.document_object["SPDXID"] = self.spdx_id(self.document.spdx_id)
         self.document_object["name"] = self.document.name
 
+        unique_doc_packages = {}
+        for doc_package in self.document.packages:
+            if doc_package.spdx_id not in unique_doc_packages.keys():
+                unique_doc_packages[doc_package.spdx_id] = doc_package
+
         package_objects = []
-        for package in self.document.packages:
+        document_describes_objects = []
+        for package in unique_doc_packages.values():
             package_info_object = self.create_package_info(package)
             # SPDX 2.2 says to omit if filesAnalyzed = False
             if package.files:
                 package_info_object["files"] = self.create_file_info(package)
-            package_objects.append({"Package": package_info_object})
+            # create a list of the packages' ids for the 'documentDescribes' field
+            package_spdx_id = package_info_object.get("SPDXID")
+            document_describes_objects.append(package_spdx_id)
```

Comment:
With this we will add all packages at document level to the list of documentDescribes SPDXIDs. As far as I understood only the packages which are part of a `DESCRIBES`-Relationship should be listed here. I am not sure if the rebase will tackle this already.